uint16 motor_id
int8 motor_temp
float32 motor_current
int16 motor_speed
int32 motor_pos